Source/CombatExtended/CombatExtended/Comps/HediffComp_Stabilize.cs

```diff
@@ -109,13 +106,33 @@ public override TextureAndColor CompStateIcon
             get
             {
                 if (bleedModifier < 1 && !parent.IsPermanent() && !parent.IsTended())
-                {
-                    return new TextureAndColor(StabilizedIcon, Color.white);
+                { //Change icon color with stabilization quality: brown - bad, yellow - good. 
+                    Color color = Color.Lerp(HediffComp_TendDuration.UntendedColor, new ColorInt(255, 212, 136).ToColor, Mathf.Clamp01(1 - bleedModifier));
```

Comment:
Any reason why we aren't using the vanilla white for 100% quality stabilisation?

The different icon should be clear enough as is to tell them apart and that way we'll be keeping the UI more consistent by sticking to the existing color scheme